apps/pxweb2/src/app/app.tsx

```diff
@@ -484,7 +495,7 @@ export function App() {
               />
             )
         )}
-    </div>
+    </div>   
```

Comment:
We should remove these empty spaces here.

Reply:
Removed as suggested